tools/oss-check

```diff
@@ -143,20 +139,7 @@ def setup_repos
   @repos.each do |repo|
     dir = "#{@working_dir}/#{repo.name}"
     puts "Cloning #{repo}"
-    if File.exists?(repo.cached_tarball)
-      perform("tar -xzvf #{repo.cached_tarball} #{dir} 2> /dev/null")
-      Dir.chdir(dir) do
-        commit_before_pull = `git rev-parse HEAD`.strip
-        perform("git pull 2> /dev/null")
-        commit_after_pull = `git rev-parse HEAD`.strip
-        if commit_before_pull != commit_after_pull
-          perform("tar -czvf #{repo.cached_tarball} #{dir} 2> /dev/null")
```

Comment:
Now I'm wondering if this should create a tarball to a temporary file and then move it atomically to prevent any interruptions from producing a corrupt tarball in the final destination.